.github/workflows/main.yml

```diff
@@ -41,8 +41,9 @@ jobs:
       - name: ▶️ Run Jest SSR
         run: npm run jest-ssr
 
-      - name: ▶️ Run Jest Screenshot
-        run: npm run jest-screenshot
+      # Temporarily disable this until after button redesign project
```

Comment:
We are also running `jest-screenshot` in the `test` script as well. I think we would want to remove it from there as well.

Reply:
Good point!! I'll update this